packages/@dcl/ecs/etc/ecs.api.md

```diff
@@ -68,25 +68,476 @@ export type ComponentDefinition<T extends ISchema = ISchema<any>> = {
 // @public (undocumented)
 export namespace Components {
     const // (undocumented)
-    Transform: ComponentDefinition<ISchema<    {
-    position: {
-    x: number;
-    y: number;
-    z: number;
-    };
-    rotation: {
-    x: number;
-    y: number;
-    z: number;
-    w: number;
-    };
-    scale: {
-    x: number;
-    y: number;
-    z: number;
+    Transform: {
```

Comment:
this types are weird, maybe the spread operator ?

Reply:
or nominal types